packages/builder/src/components/integration/QueryViewerBindingBuilder.svelte

```diff
@@ -45,12 +47,15 @@
       bindings={[...userBindings]}
       bindingDrawerLeft="260px"
       on:change={e => {
-        queryBindings = e.detail.map(binding => {
-          return {
-            name: binding.name,
-            default: binding.value,
-          }
-        })
+        dispatch(
```

Comment:
NAB: Here you can simplify the change event handling here by replacing it with just `on:change`. Then you can push your update behaviour up to the `QueryViewer` on:change event.

Reply:
Nice suggestion - that's cleaner now!